source/index.html.md

```diff
@@ -1344,83 +1344,171 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
+  </dd>
   <dt>
     travel[lodging][no_show]
     <span class="type">(true|false)</span>
   </dt>
   <dd>
-    An indicator that the cardholder is being billed for a reserved room that was not actually
-    rented (AKA a "no-show" charge). The issuer may provide this information on the customer
-    statement or use it to enhance customer service information.
+    Indicate that the cardholder is being billed for a reserved room that was
+    not actually rented (also known as a "no-show" charge).
+    The issuer may provide this information on the customer statement or use it
+    to enhance customer service information.
+    <div class="type">Optional.</div>
+  </dd>
+</dl>
+
+Additionally, if the capture is due to extra charges related to lodging, these
+should be specified using the following parameters.
+
+<dl class="dl-vertical">
+  <dt>
+    travel[lodging][extra_charge][restaurant]
+    <span class="type">(true|false)</span>
+    </dt>
+  <dd>
+    <div class="type">Optional.</div>
+  </dd>
+  <dt>
+    travel[lodging][extra_charge][gift_shop]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    <div class="type">Optional.</div>
   </dd>
   <dt>
-    travel[lodging][extra_charge]
-    <span class="type">(<code>no_extra_charge</code>|<code>restaurant</code>|<code>gift_shop</code>|<code>mini_bar</code>|<code>telephone</code>|<code>other</code>|<code>laundry</code>)</span>
+    travel[lodging][extra_charge][mini_bar]
+    <span class="type">(true|false)</span>
   </dt>
   <dd>
-    This field may be used to indicate the type of additional charges added to a customer's bill
-    after check-out. Each position in the field can be used to indicate a type of charge. If there
-    are less than six additional charges, they must be left-justified, space-filled or zero-filled
-    to the right.
+    <div class="type">Optional.</div>
   </dd>
   <dt>
-    travel[lodging][check_in]
-    <span class="type">[0-9]{6}</span>
+    travel[lodging][extra_charge][telephone]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    <div class="type">Optional.</div>
+  </dd>
+  <dt>
+    travel[lodging][extra_charge][laundry]
+    <span class="type">(true|false)</span>
+  </dt>
+  <dd>
+    <div class="type">Optional.</div>
+  </dd>
+  <dt>
+    travel[lodging][extra_charge][other]
+    <span class="type">(true|false)</span>
   </dt>
   <dd>
-    The date the customer checked into the hotel, or in the case of a no-show or an advance lodging,
-    the scheduled arrival date. The entry must be a six-digit numeric, either zeros or a valid date,
-    in the format YYMMDD.
+    <div class="type">Optional.</div>
   </dd>
 </dl>
 
-For service type Car Rental the following fields are relevant
+For service type `mobility` (often also known as "Car Rental"), the following
+fields are relevant.
 
 <dl class="dl-vertical">
   <dt>
-    travel[car][no_show]
+    travel[mobility][pick_up]
```

Comment:
```suggestion
    travel[mobility][check_out]
```
?

Reply:
```suggestion
    travel[car][pick_up]
```
tbh I think this aligns more with the naming in OP's visa/mastercard